cli/daemon/daemon.go

```diff
@@ -204,6 +204,11 @@ func (s *Server) availableUpdate() string {
 	return ""
 }
 
+var errServiceNotFound = (func() error {
```

Comment:
Can we make this `database not found` instead? Both in the identifier here as well as the error message.

Finally, we need to check for this error on the client side, using https://pkg.go.dev/google.golang.org/grpc/status#Code to check for `codes.NotFound`, and print a helpful error message there.